gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -427,6 +427,17 @@ message Alert {
   // Full description for the alert as plain-text. The information in the
   // description should add to the information of the header.
   optional TranslatedString description_text = 11;
+  
+  // Severity of this alert. This field is still experimental, 
+  // and subject to change. It may be formally adopted in the future.
```

Comment:
This field needs to be marked as experimental in the .proto file too.

Reply:
Whoops, I see that you did include this.  Could you move the "experimental" comment to it's own line to match the others?